website/assets/js/pages/osquery-table-details.page.js

```diff
@@ -35,6 +35,18 @@ parasails.registerPage('osquery-table-details', {
 
   },
   mounted: async function() {
+
+    // Check the URL to see if a platformFilter was provided.
+    if(window.location.search) {
+      // https://caniuse.com/mdn-api_urlsearchparams_get
+      let possibleSearchParamsToFilterBy = new URLSearchParams(window.location.search);
+      let platformToFilterBy = possibleSearchParamsToFilterBy.get('platformFilter');
```

Comment:
just out of curiosity: is using `let` instead of `const` a convention in the website?